convertigo/content.md

```diff
@@ -0,0 +1,98 @@
+# What is Convertigo Mobility Platform ?
+
+Convertigo Community edition is an open source MBaaS (Mobile Back end as a Service) combined with a MADP (Mobile application development platform). The platform is used to build complex Cross-platform Enterprise Mobile apps in a few days. Convertigo platform is composed of several components:
+
+1.	**Convertigo**: The back-end MBaaS server part. Runs as a Docker container with **convertigo** image
+2.	**Convertigo Web Connector**: An optional HTML web Connector server able to connect to any HTML based legacy app. Runs as a Docker container with a **web-connector** tag
+3.	**Convertigo Studio**: Runs on a Windows or a MacOS workstation, Eclipse based IDE, used to program MBaaS workflows and to build Mobile apps UIs. Can be directly downloaded from [Sourceforge.net](https://sourceforge.net/projects/convertigo/files/latest/download)
+4.	**Convertigo SDKs**: Can be used with third party Mobile development tools such as Xcode (iOS) Android Studio (Android) and Visual Studio (Windows Mobile, Windows UWP and Xamarin). SDKS are available on each platform standard repository (Bintray for Android, Cocoapods for iOS and Nuget for .NET)
+
+Convertigo Community edition brought to you by Convertigo SA (Paris & San Francisco). The platform is currently used by more than 100K developers worldwide, building enterprise class mobile apps.
+
+> [www.convertigo.com](https://www.convertigo.com)
+
+%%LOGO%%
+
+# How to use this image
+
+## Quick start
+
+	$ docker run --name C8O -d -p 28080:28080 convertigo
+
+This will start a container running the minimum Convertigo MBaaS server. Convertigo MBaaS uses images' **/workspace** directory to store configuration file and deployed projects as an Docker volume.
+
+You can access the Server admin console on http://[dockerhost]:28080/convertigo and login using the default credentials: admin / admin
+
+## Link Convertigo to a CouchDB database for FullSync (Convertigo EE only)
+
+Convertigo MBaaS FullSync module uses Apache CouchDB 1.6.1 as NoSQL repository. You can use the **[couchdb](https://hub.docker.com/_/couchdb/)** docker image and link to it convertigo this way
+
+Launch CouchDB container and name it 'fullsync'
+
+	docker run -d --name fullsync couchdb:1.6.1
+
+Then launch Convertigo and link it to the running 'fullsync' container. Convertigo MBaaS sever will automatically use it as its fullsync repository.
+
+	docker run -d --name C8O-MBAAS --link fullsync:couchdb -p 28080:28080 convertigo
+
+## Link Convertigo to a Billing & Analytics database
+
+### MySQL
+
+MySQL is the recommended database for holding Convertigo MBaaS server analytics. You can use this command to run convertigo and link it to a running MySQL container. Change [mysql](as the container name), [data base admin user], [data base admin user] with the values for your MySQL configuration.
```

Comment:
"data base admin user" is there twice :wink:. It might be clearer to do something like the following to clarify the mysql alias vs the mysql container name and provide parallels from the description to command.
```diff
- Change [mysql](as the container name), [data base admin user], [data base admin user] with the values for your MySQL configuration.
+ Change `[mysql-container]` to the container name, and `[username for the c8oAnalytics db]`, `[password for specified db user]` with the values for your MySQL configuration.

-	docker run -d --name C8O-MBAAS --link [mysql]:mysql -p 28080:28080                                   \
+	docker run -d --name C8O-MBAAS --link [mysql-container]:mysql -p 28080:28080                                   \
	    -e JAVA_OPTS="-Dconvertigo.engine.billing.enabled=true                                           \ 
	            -Dconvertigo.engine.billing.persistence.jdbc.username=[username for the c8oAnalytics db] \
-	            -Dconvertigo.engine.billing.persistence.jdbc.password=[password for the c8oAnalytics db] \
+￼	            -Dconvertigo.engine.billing.persistence.jdbc.password=[password for specified db user] \
￼
```